client/src/ratings_reviews/ReviewsList.jsx

```diff
@@ -10,115 +12,149 @@ class ReviewsList extends React.Component {
       reviewsList: [],
       shortList: [],
       allReviewsDisplayed: false,
-      sortOption: 'relevant'
-    }
+      sortOption: "relevant",
+      openModal: false
+    };
     this.handleChange = this.handleChange.bind(this);
-    this.handleClick = this.handleClick.bind(this);
+    this.handleMoreReviewsClick = this.handleMoreReviewsClick.bind(this);
+    this.openNewReviewClick = this.openNewReviewClick.bind(this);
+    this.closeNewReviewClick = this.closeNewReviewClick.bind(this);
   }
 
   componentDidUpdate(prevProps) {
     if (this.props.currentProductId !== prevProps.currentProductId) {
       helperFunction.getAllReviews(this.props.currentProductId)
         .then((response) => {
-            let allReviews = response.data.results;
-            this.setState({
-              reviewsList: allReviews.slice(0, 2)
-            })
-          })
-        .catch(err => {
-          console.log(err);
+          let allReviews = response.data.results;
+          this.setState({
+            reviewsList: allReviews.slice(0, 2),
+          });
         })
+        .catch((err) => {
+          console.log(err);
+        });
     }
   }
 
   handleChange(e) {
     let newSort = e.target.value;
     this.setState({
-      sortOption: newSort,
-    })
+      sortOption: newSort
+    });
     helperFunction.getAllReviewsWithSort(this.props.currentProductId, newSort)
       .then((response) => {
         let allReviewsWithSort = response.data.results;
         if (this.state.allReviewsDisplayed) {
           this.setState({
             reviewsList: allReviewsWithSort
-          })
+          });
         } else {
           this.setState({
             reviewsList: allReviewsWithSort.slice(0, 2)
-          })
+          });
         }
       })
-      .catch(err => {
+      .catch((err) => {
         console.log(err);
-      })
+      });
   }
 
-  handleClick(e) {
+  handleMoreReviewsClick(e) {
     e.preventDefault();
     helperFunction.getAllReviews(this.props.currentProductId)
-        .then((response) => {
-            let allReviews = response.data.results;
-            this.setState({
-              reviewsList: allReviews,
-              allReviewsDisplayed: true
-            })
-          })
-        .catch(err => {
-          console.log(err);
-        })
+      .then((response) => {
+        let allReviews = response.data.results;
+        this.setState({
+          reviewsList: allReviews,
+          allReviewsDisplayed: true
+        });
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  openNewReviewClick(e) {
+    console.log("Open Modal clicked");
```

Comment:
please remove console logs when pushing to master, so everyone has clean console :)

Reply:
Thank you for reminding me! I just removed all console logs. Thank you Luna!! 😄